community-operators/bundles/metering/bundle.0.13.0.yaml

```diff
@@ -0,0 +1,551 @@
+---
+# Source: operator-artifacts/templates/metering.yaml
```

Comment:
Should we leave this comment here? It's being added during bundle generation, I think it's fine for now as long as it doesn't cause any confusion.